### PR TITLE
Allways pick flowers from pots

### PR DESCRIPTION
**Pull request by Serzbiedry**
sorry for new branch :((((

### CODE REVIEW

File: src/main/java/carpetaddonsnotfound/mixins/FlowerPotBlock_PickFlowersFromPotsMixin.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                     BlockPos pos,
+                     PlayerEntity player,
+                     Hand hand,
+                     BlockHitResult hit,
+                     CallbackInfoReturnable<ActionResult> cir) {
+    Block potBlock = state.getBlock();
+    Block content;
+    if (!CarpetAddonsNotFoundSettings.alwaysPickFlowersFromPots || !(potBlock instanceof FlowerPotBlock) ||
+        (content = ((FlowerPotBlock) potBlock).getContent()) == Blocks.AIR) {
+      cir.cancel();
```
Comment (Gilly7CE) [inline src/main/java/carpetaddonsnotfound/mixins/FlowerPotBlock_PickFlowersFromPotsMixin.java:41]: I’m sorry to keep bringing it up but when I said this wasn’t needed I meant the entire bit with `cir` wasn’t needed in this if statement.

**Review by Gilly7CE — CHANGES_REQUESTED** [review]
Really good stuff. Just the one comment (sorry).

File: src/main/java/carpetaddonsnotfound/mixins/FlowerPotBlock_PickFlowersFromPotsMixin.java (hunk 1 above)
Comment (Serzbiedry) [reply src/main/java/carpetaddonsnotfound/mixins/FlowerPotBlock_PickFlowersFromPotsMixin.java:41]: Wait but so i removed necesarry part of code and it still worked?
Get the cross!

**Review by Gilly7CE — APPROVED** [review]
Thanks for the changes! Good work 😄

**Pull request merged**